client/src/utils.rs

```diff
@@ -60,24 +62,28 @@ pub fn get_councillors(api: &Api) -> Result<Vec<AccountId>> {
 		.ok_or_else(|| ApiClientError::Other("Couldn't get councillors".into()))
 }
 
-pub fn send_and_wait_for_in_block<C: Encode>(
+pub fn send_and_wait_for_in_block<C: Encode + Clone>(
 	api: &Api,
 	xt: EncointerXt<C>,
 	tx_payment_cid: Option<&str>,
 ) -> Option<H256> {
-	send_xt_hex_and_wait_for_in_block(api, xt.hex_encode(), tx_payment_cid)
+	send_xt_hex_and_wait_for_in_block(api, xt, tx_payment_cid)
 }
 
-pub fn send_xt_hex_and_wait_for_in_block(
+pub fn send_xt_hex_and_wait_for_in_block<C>(
 	api: &Api,
-	xt_hex: String,
+	xt: EncointerXt<C>,
 	tx_payment_cid: Option<&str>,
-) -> Option<H256> {
-	ensure_payment(api, &xt_hex, tx_payment_cid);
-	let tx_hash = api.send_extrinsic(xt_hex, XtStatus::InBlock).unwrap();
+) -> Option<H256>
+where
+	C: Clone + Encode,
```

Comment:
Why do we need the `Clone`?

Reply:
Not needed, it was here because I was messing with the extrinsic type, and at one point I needed it to be cloneable. :)